drivers/bridge/bridge.go

```diff
@@ -466,9 +474,7 @@ func parseNetworkGenericOptions(data interface{}) (*networkConfiguration, error)
 		config = opt
 	case map[string]interface{}:
 		config = &networkConfiguration{
-			EnableICC:          true,
-			EnableIPTables:     true,
-			EnableIPMasquerade: true,
```

Comment:
EnableIPMasquerade is still part of network config.
If it does not fail the test, can we retain it here.